lib/src/components/NxCloseButton/NxCloseButton.tsx

```diff
@@ -11,14 +11,12 @@ import classnames from 'classnames';
 import Close from '../../icons/Close';
 import NxButton from '../NxButton/NxButton';
 
-import './NxCloseButton.scss';
-
 const NxCloseButton = forwardRef<HTMLButtonElement, ButtonHTMLAttributes<HTMLButtonElement>>(
     function NxCloseButton({ className, ...otherProps }, ref) {
-      const btnClasses = classnames('nx-btn--close', className);
+      const btnClasses = classnames('nx-btn--icon-only', className);
```

Comment:
This is redundant, the `iconOnly` prop to `NxButton` adds this already.

Reply:
> This is redundant, the `iconOnly` prop to `NxButton` adds this already.

Right: https://github.com/sonatype/sonatype-react-shared-components/pull/138/commits/f912877a9dea22625b93362c0f3a59997b4267cd